charts/fluid/fluid/crds/data.fluid.io_databackups.yaml

```diff
@@ -1,13 +1,32 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
I think you should build with controller generator  with 0.7.0+ .

Reply:
I follow the instruction to use 0.3.0 ... 
I change it back to 0.7.0 now.